src/styles/variables.scss

```diff
@@ -0,0 +1,113 @@
+@import '~vuetify/src/styles/styles.sass';
+
+// (base: https://github.com/vuetifyjs/vuetify/blob/v2.1.18/packages/vuetify/src/styles/settings/_dark.scss)
+// NOTE: map-deep-merge() is gone.
+$material-dark: (
+        'status-bar': (
```

Comment:
You only need to include values you want to change, everything else will be merged in automatically.

Reply:
Thank you very much for your suggestion. Actually, this is my first to use SCSS directly. I didn't know the merge functionality. I'll find what color I want to change.